contracts/solidity/eligibility/NFTXENSMerkleEligibility.sol

```diff
@@ -11,14 +11,19 @@ abstract contract ENS {
 
 
 /**
- * @title NFTX Merkle Eligibility
+ * @title NFTX ENS Merkle Eligibility
  * @author Twade
  * 
- * @notice Allows vaults to be allow eligibility based on a predefined merkle tree.
+ * @notice Allows vaults to be allow eligibility based ENS domains, allowing for minimum
+ * expiration times to be set.
  */
 
 contract NFTXENSMerkleEligibility is NFTXMerkleEligibility {
 
+    /// @notice Minimum expiration time of domain
+    uint minExpirationTime;
```

Comment:
would specify this in seconds. Does this default to `public`?

Reply:
Yeah lets make it public and rename to `uint public minExpirationSeconds;`